cluster-sync/k8s-1.16.2/ceph_sc.yaml

```diff
@@ -0,0 +1,33 @@
+apiVersion: storage.k8s.io/v1
```

Comment:
I thought that we wanted to use nfs as listed in ephemeral provider. I am not sure whether we need to add more storageclasses.

Reply:
Right, I'll remove others.